pkg/fileutil/files.go

```diff
@@ -236,3 +240,68 @@ func CreateFile(p string) (*os.File, error) {
 	}
 	return os.Create(p)
 }
+
+func UploadFile(filePath, targetURL, formFileFieldName string) error {
+	file, err := os.Open(filePath)
+	if err != nil {
+		return fmt.Errorf("error opening file: %w", err)
+	}
+	fmt.Println("File opened successfully")
+	defer file.Close()
+
+	// Create a new buffer to store the multipart/form-data request
+	body := &bytes.Buffer{}
+	writer := multipart.NewWriter(body)
+
+	// Create a form field for the file
+	fileWriter, err := writer.CreateFormFile(formFileFieldName, filePath)
+	if err != nil {
+		return fmt.Errorf("error creating form file: %w", err)
```

Comment:
I have tried multiple ways, but the only way to get an error from this function call was to provide an invalid path. But then, the flow would have never reached here due to line 245. Can we leave this line without test?